CMakeLists.txt

```diff
@@ -568,7 +567,7 @@ endfunction()
 # builds.
 if(NOT OPENSSL_NO_ASM AND CMAKE_OSX_ARCHITECTURES)
   list(LENGTH CMAKE_OSX_ARCHITECTURES NUM_ARCHES)
-  if(NOT ${NUM_ARCHES} EQUAL 1)
```

Comment:
why remove encapsulation?

Reply:
Didn't get to review this commit, but they give the reason here: https://github.com/google/boringssl/commit/7e265971c0b9188261fe6e2b7526535e95d9badc